### PR TITLE
Make soundtrack state configurable, closes #5

### LINKED ISSUE #5: Sound loop breaks without user input
During development I realized that sometimes Chrome would not play the audio correctly, giving the following error on console:

```The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page.```

This happens because we are currently playing the audio without a effective way of giving the option to our users of knowing that a track will be played on that website.

https://github.com/processing/p5.js-website/issues/396
https://developers.google.com/web/updates/2017/09/autoplay-policy-changes#webaudio

We must change the behavior of the sketch to only play audio after some kind of user input.

### PR DESCRIPTION
**Pull request by dmrib**
This PR solves the Chrome issues when looping the sketch soundtrack and includes a method for controlling its state.

Pure magic 🎉

### CODE REVIEW

File: src/sketch.js [hunk 1]
```diff
@@ -2,21 +2,68 @@
 * DEFINITIONS
 */
 
+// sound controller container
+let sound = false;
+
```
Comment (anothergoldenmonkey) [inline src/sketch.js]: Can we have a more "boolean like" name, as `is_sound_on`?

File: src/sketch.js [hunk 2]
```diff
@@ -2,21 +2,68 @@
...
+ *  undefined.
+ */
+function mouseClicked()
+{
+    // soundtrack icon was clicked: update soundtrack state
+    if(mouseX > WIDTH - 50  && mouseX < WIDTH && mouseY > HEIGHT - 50 && mouseY < HEIGHT)
+    {
+        sound = !sound;
+        sound ? soundtrack.loop() : soundtrack.stop();
+    }
```
Comment (anothergoldenmonkey) [inline src/sketch.js]: As we still haven't implemented buttons properly, why not make the whole viewport toggle this action? Codewise would be cleaner than hardcoding the icon area.

**Review by anothergoldenmonkey — COMMENTED** [review]
Just left some small comments. Not super important, but I'll wait your feedback on those. In general, pretty nice. I like the keyboard, by the way. 🎹

File: src/sketch.js (hunk 2 above)
Comment (dmrib) [reply src/sketch.js]: As the webcam icon will be coming today too, we decided to keep it this way.

**Review by anothergoldenmonkey — APPROVED** [review]
🎸

**Pull request merged**